index.js

```diff
@@ -306,4 +306,22 @@ Notifications.clearAllNotifications = function() {
 	return this.callNative('clearAllNotifications', arguments)
 }
 
+Notifications.removeAllDeliveredNotifications = function() {
+  if ( Platform.OS === 'ios' ) {
+    return this.callNative('removeAllDeliveredNotifications')
+  }
+}
+
+Notifications.removeDeliveredNotifications = function() {
+  if ( Platform.OS === 'ios' ) {
+    return this.callNative('removeDeliveredNotifications', arguments)
+  }
+}
+
+Notifications.getDeliveredNotifications = function() {
+  if ( Platform.OS === 'ios' ) {
+    return this.callNative('getDeliveredNotifications', arguments)
+  }
+}
+
 module.exports = Notifications;
```

Comment:
Need to add docs for all 3 methods.  Can we get android implementations too? If not the docs need to reflect this.

Reply:
Is this waiting on anything in particular?  

Would it be too late to update this PR to handle `willPresentNotification`?